src/fitnesse/util/partitioner/MapBasedListPartitioner.java

```diff
@@ -21,7 +23,8 @@ public MapBasedListPartitioner(Function<T, String> keyFunction, Map<String, Inte
                                  BiFunction<List<List<T>>, List<T>, List<List<T>>> notFoundFunction) {
     super(
       t -> {
-        String key = keyFunction.apply(t);
+        Object object = keyFunction.apply(t);
```

Comment:
Is this change needed? It doesn't look like you made any changes to production code that require this.
If it is necessary the type of the parameter also needs to change. If you just need it for tests you can probably incorporate the handling of `WikiPage` results into the function you pass in, so this code always gets a `String` from the keyFunction

Reply:
Thanks for the suggestion. I incorporated it now. Please review.